### DISCUSSION TITLE
target=_parent for ui.link

### DISCUSSION THREAD

**Discussion by aersam** [zauberzeug/nicegui#1423] [Ideas / Feature Requests]
Hi guys!

would be nice to be able to set the new_tab parameter of ui.link to a string that's just directly the target attribute of the `a` element. We would need to set it to _parent (yes, we are still using iframes ;) )

Could also do a PR, if you want

**Comment by falkoschindler** [comment]
Hi @aersam, thanks for the suggestion!

I'm not sure about allowing `new_tab` to be a string. `ui.link(..., new_tab='_parent')` reads a bit strange.

But maybe we don't need to change the API at all. Like with every other element you can set HTML attributes using the `props` method. So you can write:

```py
ui.link('Google', 'https://google.com').props('target=_parent')
```

**Reply by aersam** [reply]
Ah makes, sense! I'll close it for now
